### ISSUE TITLE
Domain based blocking

### ISSUE THREAD

**Issue by allmightyasian** [Gedsh/InviZible#222]
Please add domain based blocking in firewall,
Like other apps, netgaurd, trackercontrol, rethink, 
Again its your choice,
For example firewall monitoring all internal traffic and user can block and unblock domain individually,

**Comment by Gedsh** [comment]
Thanks. This feature is planned.

**Comment by allmightyasian** [comment]
> Thanks. This feature is planned.

When

**Comment by Gedsh** [comment]
When the time comes. I am currently working on other features. Please be patient.

**Comment by allmightyasian** [comment]
> When the time comes. I am currently working on other features. Please be patient.

Sorry i wasn't asking for it right away,. I was just asking if you have a schedule date, like this year or next year

**Comment by Gedsh** [comment]
No problem. I don't have a specific date as this feature has a low priority. InviZible is not a firewall. This is an auxiliary feature.

**Comment by markokocic** [comment]
Blacklist Domain-based blocking now works. I guess this ticket can be closed.

**Comment by Gedsh** [comment]
@markokocic There is still no per-app domain-based blocking. There is only a DNSCrypt option for all apps.

**Comment by markokocic** [comment]
ah, ok, the ticket description didn't make it clear it's about per app settings.

I would welcome very much the ability to have this one added. It's the only reason why I still switch between Netguard and Invizible.
